app/controllers/posts_controller.rb

```diff
@@ -6,5 +6,20 @@ def index
 
   def show
     @post = Post.find_by_id(params[:id])
+    @user = User.find_by_id(params[:user_id])
+  end
+
+  def new
+    @user = User.find_by_id(params[:user_id])
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.create(author_id: params[:user_id], title: params[:title], text: params[:text])
```

Comment:
- [ ] <b>REQUIRED:</b> Here is also the same. it is really recommend to pass parameters securely.

```ruby
def create
    @post = @current_user.posts.new(post_params)
    if post.save
      #something
    else
      #something
    end
end

private 
def post_params
   params.require(:post).permit(:title, :text)
end
```